### ISSUE TITLE
pre-trained model

### ISSUE THREAD

**Issue by aiyb1314** [CuiRuikai/Partial2Complete#5]
Thank you for this amazing work! How can we run our own dataset with a pre-trained model

**Comment by aiyb1314** [comment]
What I mean is can I just input a partial point cloud and output the full point cloud?

**Comment by CuiRuikai** [comment]
Both our training and testing data is in npy format. To test your own data, you could first normalize your point cloud in the range of [-1, 1], then store then as N x 3 array and optionally save as .npy file.

Then, load the N x 3 point cloud array, and load the pre-trained model, with `tools/builder.py`.
More specifically, there are two functions, `model_builder` and `load_model`, you could load a pretrained model's config file using `EasyDict` and `yaml.load`. 

Let me know if you need more details.

**Comment by aiyb1314** [comment]
For example, I want to use EPN3D pre-trained weight parameters to complement a point cloud in our dataset what is the process?
